co2eq/purchase/index.js

```diff
@@ -10,11 +10,27 @@ import {
   UNIT_ITEM,
   UNIT_LITER,
 } from '../../definitions';
-import { convertToEuro, getAvailableCurrencies } from '../../integrations/utils/currency/currency';
 import { getChecksum } from '../utils';
 import footprints from './footprints.yml';
 import consumerPriceIndex from './consumerpriceindices.yml';
 
+const exchangeRates2011 = require('./exchange_rates_2011.json');
+
+export function convertTo2011Euro(amount, currency) {
+  if (currency.toUpperCase() === 'EUR') return amount;
+
+  const exchangeRate2011 = exchangeRates2011.rates[currency.toUpperCase()];
```

Comment:
I suggest checking the existence of the key before accessing the property, it will throw an exception for not present key anyways. Like so

```javascript
if !(currencyUpperCase in Object.keys(exchangeRates2011.rates)) {
  throw new Error(`...`)
}
const exchangeRate2011 = ...
return amount / exchangeRate2011
```